network/endpoint_windows.go

```diff
@@ -544,3 +621,103 @@ func (epInfo *EndpointInfo) GetEndpointInfoByIPImpl(ipAddresses []net.IPNet, net
 	}
 	return epInfo, errors.Wrapf(err, "No HNSEndpointID matches the IPAddress: "+ipAddresses[0].IP.String())
 }
+
+// Get PnP Device ID
+func GetPnPDeviceID(instanceID string, plc platform.ExecClient) (string, error) {
```

Comment:
all these functions can start with small letter since its not outside of this package